sdk/translation/azure-ai-translation-document/pom.xml

```diff
@@ -0,0 +1,97 @@
+<!--
+ ~ Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+Code generated by Microsoft (R) TypeSpec Code Generator.
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+    <relativePath>../../parents/azure-client-sdk-parent</relativePath>
+  </parent>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-ai-translation-document</artifactId>
+  <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-ai-translation-document;current} -->
+  
+  <name>Microsoft Azure SDK for DocumentTranslation</name>
+  <description>This package contains Microsoft Azure DocumentTranslation client library.</description>
+  <url>https://github.com/Azure/azure-sdk-for-java</url>
+
+  <licenses>
+    <license>
+      <name>The MIT License (MIT)</name>
+      <url>http://opensource.org/licenses/MIT</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <scm>
+    <url>https://github.com/Azure/azure-sdk-for-java</url>
+    <connection>scm:git:git@github.com:Azure/azure-sdk-for-java.git</connection>
+    <developerConnection>scm:git:git@github.com:Azure/azure-sdk-for-java.git</developerConnection>
+    <tag>HEAD</tag>
+  </scm>
+  <developers>
+    <developer>
+      <id>microsoft</id>
+      <name>Microsoft</name>
+    </developer>
+  </developers>
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+     <jacoco.min.linecoverage>0.2</jacoco.min.linecoverage>
+     <jacoco.min.branchcoverage>0.0</jacoco.min.branchcoverage>
```

Comment:
Needs to be fixed. Now that the tests are recorded and not LiveOnly.

Reply:
This is how the current coverage looks like 0.2 and 0.0, but will see what is missing and add coverage before GA. If you have any pointers on how to view coverage data, please share links.

Reply:
You can see the coverage report here - https://dev.azure.com/azure-sdk/public/_build/results?buildId=3864523&view=codecoverage-tab

Reply:
@srnagar curious why this is still 0.2 and 0.0, I do see the translation APIs being used in the tests